src/RoveComm/RoveCommHelpers.h

```diff
@@ -0,0 +1,117 @@
+/******************************************************************************
+ * @brief All the boilerplate I didn't want to put in RoveCommServer.h
+ *
+ * @file RoveCommHelpers.h
+ * @author OcelotEmpire (hobbz.pi@gmail.com)
+ * @date 2023-12-21
+ *
+ * @copyright Copyright Mars Rover Design Team 2023 - All Rights Reserved
+ ******************************************************************************/
+
+#ifndef ROVECOMM_HELPERS_H
+#define ROVECOMM_HELPERS_H
+
+#include <iostream>
+#include <string>
+
+#include "RoveCommManifest.h"
+
+/******************************************************************************

```

Comment:
All of these function and variables should probably be inside of a namespace or the functions should just be written into the class they belong to. Just having functions and variables globally scoped inside a file is not good practice.